src/main/resources/org/jenkinsci/plugins/googleplayandroidpublisher/ApkPublisher/config.jelly

```diff
@@ -15,6 +15,11 @@
     <f:textbox />
   </f:entry>
 
+  <f:entry title="${%Native debug symbol files}" field="nativeDebugSymbolFilesPattern"
```

Comment:
Please also add a help file for this:

 ./src/main/resources/org/jenkinsci/plugins/googleplayandroidpublisher/ApkPublisher/help-nativeDebugSymbolFilesPattern.html

In that file, explain what it's for, what the format should be, and this is where you can add the note about AGP 4.1 + AAB having built-in support for uploading native symbols automatically.